lib/greenhouse_api/client.rb

```diff
@@ -120,6 +144,18 @@ def request(http_method:, headers:, endpoint:, params:, body: {})
         status: response.status
       )
     end
+
+    def create_response(response, expected_status = 200)
+      if response&.status == expected_status
```

Comment:
Do you think we should simply parse response when status is either 200 and 201? We can add more status as we find out if they have parsable data.

```ruby
if [200, 201].include?(response&.status)
```

Would this change allow us to move this method back into `request` method, I think we always call the two in conjunction so it may be best to keep them coupled.